usr/share/rear/conf/default.conf

```diff
@@ -552,11 +552,30 @@ OPAL_PBA_KERNEL_CMDLINE=""
 # The following variable sets USE_SERIAL_CONSOLE for the PBA system.
 OPAL_PBA_USE_SERIAL_CONSOLE=""
 #
-# PBA debug password (empty for not using the debug shell facility).
-# If this pseudo-password is entered when the PBA asks for a password to unlock disks,
-# an interactive emergency shell will be started.
-# Used to debug the PBA system.
-OPAL_PBA_DEBUG_PASSWORD=""
+# OPAL_PBA_UNLOCK_MODE determines the PBA's disk unlock mode, either "transient" (the default) or "permanent".
+# * "transient" mode unlocks devices until they are powered off. This is the normal behavior.
+# * "permanent" mode unlocks devices and deactivates locking until a reactivation command is issued. This mode exists
+#   as a workaround for systems which are unable to reboot after transient unlocking. Example configuration in
+#   'local.conf' or 'site.conf':
+#       if dmidecode | grep --quiet 'Product Name: ML10Gen9$'; then
+#           # An HPE ML10Gen9 Server does not start reliably after disk unlocking unless the reboot method includes a
+#           # power cycle. In the latter case, disks will only remain unlocked if "permanent" unlock mode is used.
+#           # Seen on: HPE ML10Gen9, BIOS Version: 1.13, BIOS Release Date: 12/05/2019
+#           OPAL_PBA_KERNEL_CMDLINE+=" reboot=efi"  # reboot with a power cycle
+#           OPAL_PBA_UNLOCK_MODE="permanent"  # keep disks unlocked after power cycling
+#       fi
+#   WARNING: If this mode is used, it is recommended to run 'rear opaladmin reactivate' each time the regular
+#   operating system has booted (e.g. via a systemd unit). Even then, a protection gap exists between deactivating
+#   locking by the PBA and reactivating locking by the reagular OS. If the regular boot fails or the reactivation
+#   command is not issued for some reason, devices will remain unlocked permanently.
+OPAL_PBA_UNLOCK_MODE="transient"
+#
+# PBA debug password as a salted SHA512 hash (empty for not using the debug shell facility).
+# If the debug password is entered when the PBA asks for a password to unlock disks,
+# an interactive emergency shell will be started, which can be used to debug the PBA system.
+# To generate a password hash, use the output of 'openssl passwd -6', copy its entire output
+# between single quotes and assign it to this variable.
+OPAL_PBA_DEBUG_PASSWORD=''
```

Comment:
Your `...PASSWORD=''` let me have a look at
other such places in default.conf and right now I did
https://github.com/rear/rear/commit/580c38aba9f09177afde5ebe2d03d7a2a7010622
to fix our other password settings examples.